src/ckeditor.jsx

```diff
@@ -87,24 +126,26 @@ export default class CKEditor extends React.Component {
 						this.props.onBlur( event, editor );
 					}
 				} );
-			} )
-			.catch( error => {
-				const onErrorCallback = this.props.onError || console.error;
 
-				onErrorCallback( error );
+				// The `onReady` callback should be fired once the `editor` property
+				// can be reached from the `<CKEditor>` component.
+				// Ideally this part should be moved to the watchdog item creator listeners.
+				setTimeout( () => {
+					if ( this.props.onReady ) {
+						this.props.onReady( this.editor );
+					}
+				} );
```

Comment:
At this stage, `this.editor` should always return something since `this.watchdog` exists. Could we get rid of the timeout?